src/parser/esprima_cpp/esprima.cpp

```diff
@@ -2521,6 +2521,14 @@ class Parser {
             ASTNode right = this->isolateCoverGrammar(builder, &Parser::parseExponentiationExpression<ASTBuilder>);
             expr = this->finalize(this->startNode(startToken), builder.createBinaryExpressionExponentiationNode(left, right));
         }
+        if (this->match(NullishCoalescing)) {
+            this->nextToken();
+            this->context->isAssignmentTarget = false;
```

Comment:
Should nullish operation be handled in `parseExponentiationExpression`?
Then the name of the function should also be fixed to represent its role.